Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -232,8 +232,9 @@
   - type: Sprite
     sprite: Objects/Weapons/Guns/Projectiles/projectiles_tg.rsi
     layers:
-    - state: omnilaser
+    - state: omnilaser_greyscale
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)